### ISSUE TITLE
[FEAT]: Missing subtitles in Timed Text format & Codec tx3g support, made by SVT or youtube-dl with ffmpeg

### ISSUE THREAD

**Issue by wazerstar** [nova-video-player/aos-AVP#626]
### Description

Recently found out this beautiful software, I tested play store version and latest version as of today v6.0.62

Both missing this feature or is it a bug?

Video, I could not make sample or split since that would change the way subs are built.

https://mega.nz/file/sJpkAYbR#WOSlL_ETXT8t2w0zUVHLkIAGK3d73XGq9tGw4nfOgOo

### Additional information

MediaInfo details

```
Format                                   : MPEG-4
Format profile                           : Base Media / Version 2
Codec ID                                 : mp42 (mp42/iso2/mp41)
File size                                : 861 MiB
Duration                                 : 49 min 27 s
Overall bit rate mode                    : Variable
Overall bit rate                         : 2 433 kb/s
Encoded date                             : UTC 2022-04-19 17:30:44
Tagged date                              : UTC 2022-04-19 17:30:44


Video
ID                                       : 1
Format                                   : HEVC
Format/Info                              : High Efficiency Video Coding
Format profile                           : Main@L3.1@Main
Codec ID                                 : hvc1
Codec ID/Info                            : High Efficiency Video Coding
Duration                                 : 49 min 27 s
Bit rate                                 : 2 232 kb/s
Width                                    : 1 280 pixels
Height                                   : 720 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 25.000 FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Bits/(Pixel*Frame)                       : 0.097
Stream size                              : 790 MiB (92%)
Encoded date                             : UTC 2022-04-19 17:30:44
Tagged date                              : UTC 2022-04-19 17:30:44
Color range                              : Limited
Color primaries                          : BT.709
Transfer characteristics                 : BT.709
Matrix coefficients                      : BT.709
Codec configuration box                  : hvcC

Audio
ID                                       : 2
Format                                   : AAC LC
Format/Info                              : Advanced Audio Codec Low Complexity
Codec ID                                 : mp4a-40-2
Duration                                 : 49 min 27 s
Source duration                          : 49 min 27 s
Bit rate mode                            : Constant
Bit rate                                 : 192 kb/s
Channel(s)                               : 2 channels
Channel layout                           : L R
Sampling rate                            : 48.0 kHz
Frame rate                               : 46.875 FPS (1024 SPF)
Compression mode                         : Lossy
Stream size                              : 69.0 MiB (8%)
Source stream size                       : 69.0 MiB (8%)
Title                                    : Stereo
Default                                  : Yes
Alternate group                          : 1
Encoded date                             : UTC 2022-04-19 17:30:44
Tagged date                              : UTC 2022-04-19 17:30:44

Text
ID                                       : 3
Format                                   : Timed Text
Muxing mode                              : sbtl
Codec ID                                 : tx3g
Duration                                 : 49 min 13 s
Bit rate mode                            : Variable
Bit rate                                 : 67 b/s
Stream size                              : 24.2 KiB (0%)
Language                                 : Danish
Default                                  : Yes
Forced                                   : No
Alternate group                          : 3
Encoded date                             : UTC 2022-04-19 17:30:44
Tagged date                              : UTC 2022-04-19 17:30:44


```

**Comment by wazerstar** [comment]
I see there's another thread back in 2020, but the person did not really provide any sources or information test with.

https://github.com/nova-video-player/aos-AVP/issues/366

**Comment by courville** [comment]
Closing since released in v6.1.6.
